lib/rubocop/comment_config.rb

```diff
@@ -122,6 +123,10 @@ def each_mentioned_cop
       end
     end
 
+    def comment_on_comment_line?(comment)
```

Comment:
What do you think about making this `#directive_on_comment_line?`, since we match the directive regexp inside? 🙂

Reply:
That's much better! I'll fix it.